FogCreek/Glitch-Community
# solid-revolve: cover images shouldn't repeat if user has uploaded an image

## Links
* [solid-revolve.glitch.me](https://solid-revolve.glitch.me)
* [Manuscript](https://glitch.manuscript.com/f/cases/3328798/Cover-images-should-not-repeat-if-the-user-has-uploaded-an-image)

## GIF/Screenshots:
**Before/After**
<div>
 <img width="200" src="https://user-images.githubusercontent.com/7584833/60208804-34e3c680-9827-11e9-8e28-49c05deb99ed.png"/>
 <img width="200" src="https://user-images.githubusercontent.com/7584833/60208805-34e3c680-9827-11e9-9f2f-7f33608de154.png"/>
</div>

## Changes:
Make sure the uploaded cover image covers the whole container and does not repeat.

## How To Test:
View your "Recent projects" cover on the homepage. Your cover image should not repeat. Clear your cover image or view [a team that uses the default cover image](https://solid-revolve.glitch.me/@community), the image should repeat.
